infra/k8s/update_helm_thirdparty.sh

```diff
@@ -0,0 +1,12 @@
+#/usr/bin/env bash
+
+# Updates sub-charts not managed by the testground team.
+
+thirdparty=("stable/prometheus-pushgateway"
+	    "bitnami/prometheus-operator"
+	    "bitnami/redis")
+
+for repo in "${thirdparty[@]}"
+do
+	helm pull "$repo" --untar --untardir ./testground-infra/charts/
```

Comment:
We are mixing our source code with something we pull here, which will result in not-clean repo. We should probably pull these in a `gitignored` directory.

Reply:
My intention was to eventually add these to source control so we have the ability to roll back in the event of a bad update. I hadn't intended on running update_helm_thirdparty.sh during normal installation, but just when we want to pull in updates to those charts.